CMP1903_A1_2324/Game.cs

```diff
@@ -15,7 +15,30 @@ internal class Game
          * rolls could be continous, and the totals and other statistics could be summarised for example.
          */
 
-        //Methods
+        /// <summary>  Creates three die objects, rolls them and outputs the values and their sum. </summary>
+        /// 
+        /// <returns> first die value, second die value, third die value, sum of all dice. </returns>
+
+        //Method
+        public (int, int, int, int) Main()
+        {
+            // Create 3 dice
+            Die die1 = new Die();
+            Die die2 = new Die();
+            Die die3 = new Die();
+
+            // Roll them
+            int roll1 = die1.Roll();
+            int roll2 = die2.Roll();
+            int roll3 = die3.Roll();
+
+            int sum = roll1 + roll2 + roll3; // Calculates the total value of all three dice rolls
+
+            Console.WriteLine($"Roll 1: {roll1}\nRoll 2: {roll2}\nRoll 3: {roll3}\nTotal: {sum}");
+
+            return (roll1, roll2, roll3, sum);
+        }
+
 
     }
 }
```

Comment:
Functionality for Multiple Rolls would be a good addition to increase the capability of the code.